packages/tokamak/contracts-bedrock/hardhat.config.ts

```diff
@@ -0,0 +1,33 @@
+import "@nomiclabs/hardhat-waffle";
+import "@nomicfoundation/hardhat-verify";
+import 'hardhat-deploy';
+
+export default {
+  solidity: "0.8.9",
+  networks: {
+    devnetL1: {
+      url: "http://localhost:9545",
+      chainId: 901,
+    },
+    'thanos-sepolia-test': {
+      url: "https://explorer.thanos-sepolia-test.tokamak.network",
+      chainId: 111551118080,
+    }
+  },
+  etherscan: {
+    apiKey: "YOUR_API_KEY",
+    customChains: [
```

Comment:
I think we should add thanos-sepolia-test in the customChains as well to be able to verify the network. Can you check it?

Reply:
Sure, let me check it out!